README.md

```diff
@@ -130,9 +130,9 @@ In the documentation you will see examples using [object spread properties](http
 
 ## Why?
 
-When writing styles in JavaScript, many people still need Sass-style helper functions to be productive. `✨ polished` brings them to you in a nice, lightweight package tailor-made for styles in JavaScript.
+When writing styles in JavaScript, people need Sass-style helper functions to be productive. `✨ polished` brings them to you in a nice, lightweight package tailor-made for styles in JavaScript.
```

Comment:
I think this loses the original intent of the sentence. Not everyone needs them to be productive. The better edit here would probably be `many people need Sass-style helper functions to be productive.`